src/App.vue

```diff
@@ -317,7 +323,7 @@ export default {
 
     storeAuthorizations (response: any): void {
       // NB: roles array may contain 'view', 'edit' or nothing
-      const authRoles = response && response.data && response.data.roles
+      const authRoles = response?.data?.roles
```

Comment:
nice clean up. I read should read the docs a little further on optional chaining and how it works. I sure do like doing away with the short circuiting!

Reply:
It stops evaluating when the token is null. E.g.,
null?.data.roles is null
response.null?.roles is null
response.data.null is null